cartridges/int_externalCustomerData/cartridge/scripts/hooks/requestToExternalService.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+var urlUtils = require('../utils/urlUtils');
+
+function requestCustomerToExternalService(method, registrationForm, integrationId) {
+    var url;
+    method === "POST" ? url = `${urlUtils.host}${urlUtils.customers}`
```

Comment:
It is always POST

Reply:
If a customer is editing the profile, we are sending PUT request and changing the endpoint.